pypesto/optimize/ess/sacess.py

```diff
@@ -328,6 +328,7 @@ def submit_solution(
         sender_idx: Index of the worker submitting the results.
         elapsed_time_s: Elapsed time since the beginning of the sacess run.
         """
+        abs_change = fx - self._best_known_fx.value
```

Comment:
Naming a bit misleading as you use `abs` command below. But no idea for better naming, expect perhaps `change`

Reply:
I think gets relatively clear from the context. `change` would as ambiguous - (absolute) relative or (absolute) absolute?.